debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -341,7 +341,8 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     USER,
                     PASSWORD,
                     SERVER_TIMEZONE,
-                    INSTANCE)
+                    INSTANCE,
+                    CommonConnectorConfig.SKIPPED_OPERATIONS)
```

Comment:
I noticed a comment in the PostgreSQL implementation PR that suggested this to be removed since its included in the super type's configuration.  Can we follow the same approach here?